examples/taskruns/step-script.yaml

```diff
@@ -75,3 +75,12 @@ spec:
           print('Got: ', v)
           quit()
 
+    # Test that args are allowed and passed to the script as expected.
+    - name: args-allowed
+      image: ubuntu
+      args: ['hello', 'world']
+      script: |
+        #!/usr/bin/env bash
```

Comment:
NIT: we should start adding `set -e` in our scripts, just to promote best practices 👼